git/dogfood.sh

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+featureBranch=$(git rev-parse --abbrev-ref HEAD)
+dogfoodBranch=dogfood/$featureBranch
+
+info() {
+    echo "* $@"
+}
+
+branchExists() {
+    git rev-parse -q --verify "$1" >/dev/null
+}
+
+createDogfoodBranch() {
+    git branch "$dogfoodBranch"
+}
+
+rebaseDogfoodBranch() {
+    git checkout "$dogfoodBranch"
+    git rebase "$featureBranch"
```

Comment:
does that work when the feature branch has been force-pushed ? I have the feeling that some conflicts will be raised. Should it be `git reset --hard "$featureBranch"` ?